dev-infrastructure/docs/development-setup.md

```diff
@@ -378,12 +397,12 @@ ocm login --url=http://localhost:8000 --use-auth-code
 ```bash
 NAME="<INSERT-NAME-HERE>"
 RESOURCENAME="<INSERT-NAME>"
-SUBSCRIPTION="<INSERT-NAME>"
-RESOURCEGROUPNAME="<INSERT-NAME>"
-TENANTID="<INSERT-NAME>"
+SUBSCRIPTION=$(echo $(az account subscription list | jq '.[] | select(.displayName == "ARO Hosted Control Planes (EA Subscription 1)")' | jq -r '.subscriptionId'))
+RESOURCEGROUPNAME="test-RG"
+TENANTID=$(echo $(cat azure-creds.json | jq -r '.tenantId'))
```

Comment:
I think we can keep <INSERT-NAME> here too. Even though this should work the usage of azure-creds.json is temporary so let's keep in unmodified